sources/cron.h

```diff
@@ -13,6 +13,7 @@ struct od_cron
 {
 	uint64_t     stat_time_us;
 	od_global_t *global;
+	od_atomic_u64_t startup_errors;
```

Comment:
we can make this operation non-atomic since the exact number of errors of this kind is not very important